lib/ansible/modules/windows/win_regedit.ps1

```diff
@@ -21,313 +22,317 @@
 
 $ErrorActionPreference = "Stop"
 
-$params = Parse-Args $args -supports_check_mode $true
+$params = Parse-Args -arguments $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
-$diff_support = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
+$diff_mode = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
```

Comment:
Well, it's not a mode. You only have run-mode and check-mode. The diff option influences the output (but not the behavior of how it operates). That's why it was called _ansible_diff, and not _ansible_diff_mode (like _ansible_check_mode).

If it doesn't matter, why changing it at all :-) But yes, to me it matters to do it consistently across all modules (as what we did).

```
[dag@moria ansible.git]$ grep -rl 'diff_support' lib/ansible/modules/windows/
lib/ansible/modules/windows/win_firewall_rule.ps1
lib/ansible/modules/windows/win_timezone.ps1
lib/ansible/modules/windows/win_lineinfile.ps1
lib/ansible/modules/windows/win_environment.ps1
```